clusterctl/examples/openstack/README.md

```diff
@@ -0,0 +1,24 @@
+# Openstack Example Files
+## Contents
+*.yaml files - concrete example files that can be used as is.
+*.yaml.template files - template example files that need values filled in before use.
```

Comment:
Looks like some of the items in machines.yaml.template is specific to someone's openstack environment, we should tell the users to go edit that file before running generate-yaml.sh. Also provide guidance on which fields they can edit and how to find it in their own environment would be awesome

Reply:
Another idea would be to prompt the user for this information like we do for the username/password etc.

Reply:
Yeah, that would be better, but there are so many fields that can be specified by user, prompt user for all those information can take long time. I think provide a guidance would be better. I can work on that latter.